src/xcode/ENA/ENA/Source/Views/ENSetting/ActionTableViewCell.swift

```diff
@@ -117,4 +117,8 @@ class ActionTableViewCell: UITableViewCell, ActionCell {
 		}
 	}
 
+	override func accessibilityActivate() -> Bool {
+		toggle(self)
```

Comment:
why do you call the toggle?

Reply:
I called that since that is what the accessibility custom action already calls.
But looking at it now I think we can refactor this a little bit. Let me see.